src/helpers/hardhat.ts

```diff
@@ -0,0 +1,16 @@
+import { chainRegistry } from '../assets'
+import { ChainIds, ChainName, chainNameSchema } from '../constants'
+
+export function getNetworkConfig(
+    chainName: ChainName,
+    explorerKeys?: Partial<Record<ChainName, string>>,
+) {
+    const chain = chainRegistry[ChainIds[chainNameSchema.parse(chainName)]]
```

Comment:
what about refactoring the chain registry to be a static class in the future? in this way we could add some methods to improve the readability to quickly understand what's going in this line to be something like this for example:

```ts
const chain = ChainRegistry.getChainByName(chainName)
```

Do you think it actually makes sense to refactor the chain registry or just creating a simple helper would solve this?

Reply:
To be honest I don't see many use cases for finding a chain by its name, this will only be used when configuring your hardhat environment, where you can have your .env with something like `HARDHAT_NETWORK=ARBITRUM_TESTNET` instead of having to know the exact chain id for this network.

But in all other use cases, you will already be using the `chain id` and not the name since it will be all programmatical